perllib/FixMyStreet/Cobrand/TfL.pm

```diff
@@ -467,9 +467,7 @@ around 'munge_sendreport_params' => sub {
 
     $self->$orig($row, $h, $params);
 
-    if ($row->category eq "Countdown - not working") {
-        $params->{From} = [ $self->do_not_reply_email, $self->contact_name ];
-    }
+    $params->{From} = [ $self->do_not_reply_email, $self->contact_name ];
```

Comment:
always_use_reply_to is set for all UK cobrands, so I think the Reply-To header would also need removing, as that contains the user's email.